### PR TITLE
Add TGLU configuration

### PR DESCRIPTION
**Pull request by Sae86**
Signed-off-by: Sae86 <sae.batllori@intel.com>

### CODE REVIEW

File: chipsec/cfg/8086/tglu.xml [hunk 1]
```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0"?>
+<configuration>
```
Comment (npmitche) [inline chipsec/cfg/8086/tglu.xml:2]: Missing `platform="TGLU"` in configuration tag.

File: chipsec/cfg/8086/tglu.xml [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+<!--
+CHIPSEC: Platform Security Assessment Framework
+Copyright (c) 2020-2021, Intel Corporation
```
Comment (npmitche) [inline chipsec/cfg/8086/tglu.xml:5]: Does not need to include 2020, publish date will be in 2021.

File: chipsec/cfg/8086/tglu.xml [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+* Intel(R) 500 Series Chipset Family on-package Platform Controller Hub (PCH)
+-->
+
+  <!-- #################################### -->
+  <!--                                      -->
+  <!-- Information                          -->
+  <!--                                      -->
+  <!-- #################################### -->
+  <info family="core">
+    <sku did="0x9A12" name="TigerLake" code="TGLU" longname="(TGL UP4 4 Cores)"/>
```
Comment (npmitche) [inline chipsec/cfg/8086/tglu.xml:40]: Should remove ()'s from longname

**Review by BrentHoltsclaw — APPROVED** [review]

**Review by mjkyung — APPROVED** [review]

File: chipsec/cfg/8086/tglu.xml [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+  <!--                                      -->
+  <!-- Configuration registers              -->
+  <!--                                      -->
+  <!-- #################################### -->
+  <registers>
+    <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+    <!-- MSR Definitions              -->
+    <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+    <!-- CPU Model Specific Registers -->
+
```
Comment (BrentHoltsclaw) [inline chipsec/cfg/8086/tglu.xml:89]:
```suggestion
    <register name="PRMRR_MASK" type="msr" msr="0x1F5" desc="PRMRR MASK register">
      <field name="PRMRR_LOCK" bit="10" size="1" desc = "PRMRR Lock Bit" />
      <field name="PRMRR_VLD" bit="11" size="1" desc = "PRMRR Valid Bit set by mcheck" />
      <field name="PRMRR_mask_bits"   bit="12"  size="40" desc="PRMRR mask bits" />
    </register>
    <register name="PRMRR_PHYBASE" type="msr" msr="0x2A0" desc="PRMRR BASE Address 0">
      <field name="PRMRR_MEMTYPE" bit="0" size="3" desc="PRMRR Memory Type" />
      <field name="PRMRR_CONFIGURED" bit="3" size="1" desc="PRMRR Base Configured" />
      <field name="PRMRR_base_address_fields" bit="12" size="40" desc="PRMRR base address bits" />
    </register>
    <register name="MSR_BIOS_DONE" type="msr" msr="0x151" desc="BIOS Done Status">
      <field name="SoC_BIOS_DONE" bit="1" size="1" desc="SoC init done"/>
      <field name="IA_UNTRUSTED" bit="0" size="1" desc="Untrusted mode enable bit"/>
    </register>
    <register name="MSR_SMM_FEATURE_CONTROL" type="msr" msr="0x4E0" desc="Enhanced SMM Feature Control">
      <field name="LOCK"            bit="0" size="1"  desc="Lock bit" />
      <field name="SMM_Code_Chk_En" bit="2" size="1"  desc="Prevents SMM from executing code outside the ranges defined by the SMRR" />
    </register>
```

File: chipsec/cfg/8086/tglu.xml [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+    </register>
+
+    <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+    <!--      Undefined Registers     -->
+    <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+    <register name="PCI0.0.0_MESEG_BASE" undef="MESEG mapping method has changed." />
+    <register name="PCI0.0.0_MESEG_MASK" undef="MESEG mapping method has changed." />
+    <register name="PCI0.0.0_SMRAMC" undef="Compatible SMRAM is not supported." />
+    <register name="MSR_LT_LOCK_MEMORY" undef="Not defined for platform" />
+
```
Comment (BrentHoltsclaw) [inline chipsec/cfg/8086/tglu.xml:172]:
```suggestion
    <register name="PRMRR_UNCORE_PHYBASE" undef="Not defined for the platform" />
    <register name="PRMRR_UNCORE_MASK" undef="Not defined for the platform" />   
```

**Pull request merged**